package.json

```diff
@@ -4,30 +4,37 @@
         "dev": "vite",
         "build": "vite build",
         "lint": "npm run lint:eslint",
-        "lint:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.js --cache",
+        "lint:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.ts --cache",
         "fix": "npm run fix:eslint",
-        "fix:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.js --fix",
+        "fix:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.ts --fix",
         "analyze": "cross-env ANALYZE=true npm run prod",
         "apidoc": "npm run apidoc:connect",
         "apidoc:connect": "apidoc -i src/Connect/ -c docs/src/connect-api -t docs/src/connect-api/template -o docs/dist/api/connect"
     },
+    "dependencies": {
+        "alpinejs": "^3.0.6",
+        "js-cookie": "^3.0.1"
+    },
     "devDependencies": {
         "@defstudio/vite-livewire-plugin": "^1.0.0",
         "@tailwindcss/aspect-ratio": "^0.4.0",
         "@tailwindcss/forms": "^0.5.2",
         "@tailwindcss/line-clamp": "^0.4.0",
         "@tailwindcss/typography": "^0.5.0",
-        "alpinejs": "^3.0.6",
+        "@types/alpinejs": "^3.7.1",
+        "@types/js-cookie": "^3.0.3",
+        "@typescript-eslint/eslint-plugin": "^5.53.0",
+        "@typescript-eslint/parser": "^5.53.0",
         "autoprefixer": "^10.4.7",
         "eslint": "^8.4.1",
         "eslint-config-airbnb-base": "^15.0.0",
         "eslint-plugin-import": "^2.20.2",
         "laravel-echo": "^1.4.0",
         "laravel-vite-plugin": "^0.7.0",
-        "lodash": "^4.17.19",
```

Comment:
This appeared to be unused so I took the opportunity to delete it. I am not sure when we would ever need `lodash`, but in the event we do, using a pure module for the single needed function is generally a better choice, eg: [lodash.merge](https://www.npmjs.com/package/lodash.merge). Lodash is not tree-shakeable, so a single import from lodash massively inflates the bundle size.

Reply:
It's a default dependency in the Laravel skeleton. It's fine to have it removed imo